themes/default/content/blog/pulumi-release-notes-68/index.md

```diff
@@ -0,0 +1,131 @@
+
+Learn more in our [blog post on Update Plans]({{< relref "/blog/announcing-public-preview-update-plans/" >}}) and in this [Update Plans GitHub issue](https://github.com/pulumi/pulumi/issues/2318)
+
+
+### Disable default provider
+
+While default providers are enabled by default, you can now [disable default providers]({{< relref
+"/docs/intro/concepts/config#special-configuration-options" >}}) on a per stack basis. Disabling default
+providers is a good idea if you want to ensure that your programs must be explicit about which provider they
+will use. For example, to disable the `aws` provider, you can run:
+
+```sh
+$ pulumi config set --path 'pulumi:disable-default-providers[0]' aws
+```
+
+If you wanted to also disable the `kubernetes` default provider, as well as the `aws` default provider, you could run:
+
+```sh
+$ pulumi config set --path 'pulumi:disable-default-providers[1]' kubernetes
+```
+
+This adds a new entry to the list `pulumi:disable-default-providers`. To disable all default providers, use `*` as the package name:
+
+```sh
+$ pulumi config set --path 'pulumi:disable-default-providers[0]' '*'
+```
+Learn more in the [Disable Default Providers GitHub issue](https://github.com/pulumi/pulumi/issues/3383)
+
+### `pulumi up --target` wildcard URN support
+
+Our community had requested that we add wildcard URN support to `pulumi up --target` and as a result we now support wildcards for `pulumi up --target <urn>` and similar commands. 
+
+Learn more in the [Wildcards support Github issue](https://github.com/pulumi/pulumi/issues/5870)
+
+## Pulumi Service & Pulumi.com
+
+### Sign-up and sign-in experience revamp
+
+We spent time this milestone revamping our sign-in and sign-up process in the Pulumi Service. Our intentions were to make it easier to differentiate and toggle between the sign-in and sign-up experiences. As part of this work we also refreshed the design to align with our public website. Take a look! 
+
+The new Pulumi Service sign-in page
+![A screenshot of the Pulumi Service sign-in page"](sign-in.png)
+
+The new Pulumi Service create an account page
+![A screenshot of the Pulumi Service create an account page"](sign-up.png)
+
+### Show resource URN on resource detail page
+
+We added the resource URN on the resource details page within the Pulumi Service UI. This enables users to easily find the URN for commands like `pulumi up --replace` or `--target`. Users can easily copy the resource URN to their clipboard by clicking on the copy icon.
```

Comment:
We can omit "UI" here because it's clear (by the screenshot and "resource details page") that we're talking about the console. 

```suggestion
We added the resource URN on the resource details page within the Pulumi Service. This enables users to easily find the URN for commands like `pulumi up --replace` or `--target`. Users can easily copy the resource URN to their clipboard by clicking on the copy icon.
```